11/debian-9/rootfs/setup.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#
+# Bitnami PostgreSQL setup
+
+# shellcheck disable=SC1090
+# shellcheck disable=SC1091
+
+# Load Generic Libraries
+. /libfile.sh
+. /liblog.sh
+. /libservice.sh
+. /libvalidations.sh
+
+# Functions
+
+set -o errexit
+set -o nounset
+set -o pipefail
+# set -o xtrace # Uncomment this line for debugging purpose
+# shellcheck disable=SC1091
+
+# Load libraries
+. /libfs.sh
+. /libos.sh
+. /libpostgresql.sh
+
```

Comment:
Are you loading the libraries and declaring `shellcheck disable` actions twice? Please unify it.